src/components/Components.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import { StyleSheet, View, Text } from 'react-native';
+import { Colors } from 'src/constants/Colors';
+import { FontAwesome } from '@expo/vector-icons';
+const styles = StyleSheet.create({
+  screen: {
+    backgroundColor: Colors.Background,
+  },
+  row: {
+    flexDirection: 'row',
+  },
+  pill: {
+    borderRadius: 100,
+    borderWidth: 1,
+    borderColor: Colors.Grey100,
+    padding: 10,
+    alignItems: 'center',
+  },
+});
+
+export function ScreenContainer(props: React.ComponentProps<typeof View>) {
+  return <View {...props} style={[styles.screen, props.style]} />;
+}
+
+export function RowContainer(props: React.ComponentProps<typeof View>) {
```

Comment:
im fine with creating layout components for rows and columns,  but put it in the shared components folder as `Row.tsx`